native/explorer/src/series.rs

```diff
@@ -308,8 +308,8 @@ pub fn s_len(data: ExSeries) -> Result<usize, ExplorerError> {
 #[rustler::nif(schedule = "DirtyCpu")]
 pub fn s_fill_none(data: ExSeries, strategy: &str) -> Result<ExSeries, ExplorerError> {
     let strat = match strategy {
-        "backward" => FillNullStrategy::Backward,
-        "forward" => FillNullStrategy::Forward,
+        "backward" => FillNullStrategy::Backward(None),
+        "forward" => FillNullStrategy::Forward(None),
```

Comment:
Do you know what the enum value is for here? I couldn't find it in the docs.

Reply:
So, had a dig as I was also curious!

Boils down to this issue: https://github.com/pola-rs/polars/issues/3545
Which was implemented here: https://github.com/pola-rs/polars/pull/3559

It's defined in

 https://docs.rs/polars/latest/polars/prelude/enum.FillNullStrategy.html

Reply:
Ohhhh interesting! Okay! Thanks for finding that :).